pkg/remote/resource_enumeration_error_handler.go

```diff
@@ -21,9 +21,9 @@ func HandleResourceEnumerationError(err error, alertr *alerter.Alerter) error {
 		return err
 	}
 
-	if reqerr.StatusCode() == 403 {
+	if reqerr.StatusCode() == 403 || strings.Contains(reqerr.Code(), "AccessDenied") {
```

Comment:
To be sure to catch only this case maybe we could check the status code too ?

```go
if reqerr.StatusCode() == 403 || (reqerr.StatusCode() == 400 && reqerr.Code() == "AccessDeniedException") {
```

Reply:
From what we saw AccessDeniedException is always access denied it does not depends on the http code so I remove te 400 check but no matter what we do we won't catch everything, 403 on dynamo is still another error.